shipstation_integration/public/js/tag_list.js

```diff
@@ -0,0 +1,12 @@
+frappe.listview_settings["Tag"] = {
+    refresh: function (listview) {
+        listview.page.add_inner_button("Sync Tags from ShipStation", function () {
+            frappe.call({
```

Comment:
I think this call should also be available in the Shipstation Settings doctype itself